src/qrcore.cpp

```diff
@@ -511,14 +511,14 @@ void QRCore::setDefaultCPU() {
 
 QString QRCore::assemble(const QString &code) {
     RAsmCode *ac = r_asm_massemble (core->assembler, code.toUtf8().constData());
-    QString hex = QString(ac->buf_hex);
+    QString hex(nullptr != ac ? ac->buf_hex : "");
```

Comment:
Ok i can change that. But on I hope it's ok to to 'yoda compare' no equality (==)?